security.dd

```diff
@@ -0,0 +1,27 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+If you have any security-related concerns, please don't hesitate to contact us at
+$(LINK2 mailto:security@dlang.org, security@dlang.org).
+
+$(H3 Sending encrypted messages)
+
+You can encrypt your message against the $(LINK2 $(ROOT)gpg_keys.html, D's official keyring).
```

Comment:
I went with using the release signing keys for now, because it's essentially the same team and it saves us the trouble of creating and distributing a shared gpg key.
No strong feelings here, which approach do you like the most?